### PR TITLE
chore(JumpLinks): convert examples to TypeScript

### LINKED ISSUE #7494: Convert Jump links examples to typescript

### PR DESCRIPTION
**Pull request by tompsota**
Signed-off-by: Tomas Psota <tpsota@redhat.com>

**What**: Closes #7494

### CODE REVIEW

File: packages/react-core/src/components/JumpLinks/examples/JumpLinks.md [hunk 1]
```diff
@@ -9,89 +9,26 @@ propComponents: ['JumpLinks', 'JumpLinksItem']
...
-    <JumpLinksList>
-      <JumpLinksItem isActive>Active subsection</JumpLinksItem>
-      <JumpLinksItem>Inactive subsection</JumpLinksItem>
-      <JumpLinksItem>Inactive subsection</JumpLinksItem>
-    </JumpLinksList>
-  </JumpLinksItem>
-  <JumpLinksItem>Inactive section</JumpLinksItem>
-  <JumpLinksItem>Inactive section</JumpLinksItem>
-</JumpLinks>
+```ts file='./JumpLinksVerticalWithSubsection.tsx'
```
Comment (jenny-s51) [inline packages/react-core/src/components/JumpLinks/examples/JumpLinks.md:32]:
```suggestion
```ts file='./JumpLinksExpandableVerticalWithSubsection.tsx'
```

File: packages/react-core/src/components/JumpLinks/examples/JumpLinksVerticalWithSubsection.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { JumpLinks, JumpLinksItem, JumpLinksList } from '@patternfly/react-core';
+
+export const JumpLinksVerticalWithSubsection: React.FunctionComponent = () => (
```
Comment (jenny-s51) [inline packages/react-core/src/components/JumpLinks/examples/JumpLinksVerticalWithSubsection.tsx:4]:
```suggestion
export const JumpLinksExpandableVerticalWithSubsection: React.FunctionComponent = () => (
```

File: packages/react-core/src/components/JumpLinks/examples/JumpLinks.md [hunk 3]
```diff
@@ -9,89 +9,26 @@ propComponents: ['JumpLinks', 'JumpLinksItem']
...
-```js
-import React from 'react';
-import { JumpLinks, JumpLinksItem } from '@patternfly/react-core';
-
-<JumpLinks isCentered>
-  <JumpLinksItem>Inactive section</JumpLinksItem>
-  <JumpLinksItem isActive>Active section</JumpLinksItem>
-  <JumpLinksItem>Inactive section</JumpLinksItem>
-</JumpLinks>
+```ts file='./JumpLinksCentered.tsx'
```
Comment (jenny-s51) [inline packages/react-core/src/components/JumpLinks/examples/JumpLinks.md:16]:
```suggestion
```ts file='./JumpLinksWithCenteredList.tsx'
```

File: packages/react-core/src/components/JumpLinks/examples/JumpLinksCentered.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { JumpLinks, JumpLinksItem } from '@patternfly/react-core';
+
+export const JumpLinksCentered: React.FunctionComponent = () => (
```
Comment (jenny-s51) [inline packages/react-core/src/components/JumpLinks/examples/JumpLinksCentered.tsx:4]:
```suggestion
export const JumpLinksWithCenteredList: React.FunctionComponent = () => (
```

**Review by jenny-s51 — COMMENTED** [review]
These look great @tompsota - nice work! Just a couple nits about the file and function names  in order to make them a bit clearer 🙂 

Otherwise LGTM!

**Review by kmcfaul — APPROVED** [review]

**Review by jenny-s51 — APPROVED** [review]
Looks 💯

**Pull request merged**